Moodboard-program/moodboard-prototype

```diff
@@ -0,0 +1,262 @@
+import sys
+import time
+
+
+
+def paintblack(canvas2):
+  pixels = getPixels(canvas2)
+  for i in pixels:
+    setColor(i,black)
+    
+
+def start():
+  #load image from file
+  canvas2=makePicture(r".\Ship-image-structure\ship+biodomes.jpg")
+  show(canvas2)
+  #get input from user
+  
+  keypress = requestNumber("Press 1 for biodome or 2 for ship: ")
+  #if the user entered 1 then zoom into the top section of the image
+  #if the user enters 2(or anything else) zoom into the bottom part of the image
```

Comment:
This comment contradicts the code, as the code only checks if the user has pressed 1 or 2. It doesn't do anything if they enter anything else.